### DISCUSSION TITLE
[Bug]: System Metrics and Logs options missing from settings menu on systems with higher than 100% display scaling

### DISCUSSION THREAD

**Discussion by JeffWDH** [blakeblackshear/frigate#12741] [Report a Bug] [answered]
### Describe the problem you are having

Tested versions: v0.14.0-rc2, 0.14.0-beta4
Browsers: Firefox, Chrome

When clicking on the settings menu, the system metrics and system logs options are hidden on Windows systems with display scaling set higher than 100%. In my example the screen scaling in Windows is set at 150%. Reducing the zoom in the browser will make these items visible but they are not visible at 100%. See below screenshots:

Browser zoom 100%:
![image](https://github.com/user-attachments/assets/21a3c1b7-92b1-484d-b095-c46ae22d28a9)

Browser zoom 80%:
![image](https://github.com/user-attachments/assets/7e865a6c-cef6-4377-9483-c1f02c4e7fbd)

### Steps to reproduce

- Set Windows display scaling to 150%. 
- Click settings icon in Frigate.
- Press ctrl - in browser to reduce zoom and the options will start to be visible.

### Version

v0.14.0-rc2, 0.14.0-beta4

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### Operating system

Windows

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

N/A

### Any other information that may be helpful

N/A

**Comment by hawkeye217** [comment, marked as answer]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/12743

**Reply by JeffWDH** [reply]
Thanks!
